apps/beets-frontend-v3/next.config.ts

```diff
@@ -25,12 +18,21 @@ const config: NextConfig = {
         hostname: '**',
       },
     ],
+    localPatterns: [
+      {
+        pathname: '/api/proxy/image/**',
+      },
+      {
+        pathname: '/images/**',
+      },
+    ],
     minimumCacheTTL: 60,
   },
   transpilePackages: ['@repo/lib'],
 
   // Safe App setup
   headers: manifestHeaders,
+  reactCompiler: true,
```

Comment:
would it be worthwhile / safer to activate with a separate PR after battle testing the react 16 + turbopack change? or maybe [incremental adoption](https://react.dev/learn/react-compiler/incremental-adoption)?

i don't have a strong understanding or opinion here just thoughts / suggestions

Reply:
i think the gradual adoption kicks in once we start removing all the manual memoization